app/src/main/res/layout/component_collection_creation.xml

```diff
@@ -26,17 +26,17 @@
         android:textColor="@color/fx_mobile_text_color_oncolor_primary"
         android:textSize="20sp"
         app:drawableStartCompat="@drawable/mozac_ic_back"
+        app:layout_constrainedWidth="true"
```

Comment:
Can you move this on line 30? I believe this is the correct formatting.

Reply:
Thank you for reviewing the code. Since i have i have lost access to old Git account, I have made the changes here https://github.com/mozilla-mobile/fenix/pull/25038

Reply:
Thank you for reviewing the code. That change is done.